### ISSUE TITLE
AuthenticationContextTests#testAcquireTokenUserId() doesn't use mock network resources

### ISSUE THREAD

**Issue by iambmelt** [AzureAD/azure-activedirectory-library-for-android#1002]
This test passes/fails inconsistently: it does not use mocked network resources.

Failure case:
```
com.microsoft.aad.adal.AuthenticationContextTest > testAcquireTokenUserId[test(AVD) - 5.0.2] FAILED 
    org.junit.ComparisonFailure: authority is same expected:<https://login.[windows.net]/common> but was:<https://login.[microsoftonline.com]/common>
    at org.junit.Assert.assertEquals(Assert.java:115)
:adal:connectedDebugAndroidTest FAILED
FAILURE: Build failed with an exception.
```

**Comment by iambmelt** [comment]
Issue fixed on `release/1.13.2` - not on `dev` yet (10/19/2017)

**Comment by nazukj** [comment]
@iambmelt to confirm merge and close.

**Comment by iambmelt** [comment]
Test was removed in [3688b0d617b6196f75719715dccd80ab4985b0e3](https://github.com/AzureAD/azure-activedirectory-library-for-android/commit/3688b0d617b6196f75719715dccd80ab4985b0e3#diff-1f51ca351931fc74098edcfa59eecbd3)
Change was released in `1.13.2+`
